resources/packs/oci/oci.lr

```diff
@@ -1,14 +1,14 @@
 option go_package = "go.mondoo.com/cnquery/resources/packs/oci"
 
-// Oracle Cloud Infrastructure Tenant
+// Oracle Cloud Infrastructure (OCI) Tenancy
```

Comment:
This improves search results in our internal docs search as well as via Google. Otherwise, the MQL docs pages lack the word Oracle in them.